requirements/base.in

```diff
@@ -1,5 +1,7 @@
 # Core requirements for using this package
 
+django-nose
```

Comment:
edx-platform depends on xblock-utils and currently doesn't need this; does this really need to be a core dependency of the package, or just a testing dependency?

Reply:
It seems it was no longer being used. Removed it.